pkg/tinycrypt/patches/0001-adapt-Makefiles.patch

```diff
@@ -0,0 +1,51 @@
+From a0e94f07ccf9c90ca49a905adc925b4ebfd8f0eb Mon Sep 17 00:00:00 2001
```

Comment:
I propose copying this file, overwriting the one in the package sources, as other packages (like minmea) do.
Maintaining the patch is way more overhead.